### PR TITLE
Remove scripts related to k8s-artifacts-prod-vuln-dashboard

### LINKED ISSUE #1642: Remove scripts related to k8s-artifacts-prod-vuln-dashboard
This bucket has been manually removed, but we're in a state where if someone re-runs `infra/gcp` scripts, it will get re-provisioned.  Please open a PR to remove from the scripts

/milestone v1.21
/priority critical-urgent
We're blocked from running scripts to roll out any further changes until this is done
/wg k8s-infra
/sig release
/area release-eng
/assign @justaugustus 
or please find a @kubernetes/release-engineering member capable of making the PR

### PR DESCRIPTION
**Pull request by hasheddan**
This bucket has been manually removed but these scripts will re-create
it if not cleaned up.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

Fixes #1642 

/assign @spiffxp @justaugustus

### CODE REVIEW

**Comment by justaugustus** [comment]
For RelEng:
/lgtm
/approve

File: infra/gcp/ensure-prod-storage.sh [hunk 1]
```diff
@@ -372,44 +372,6 @@ color 6 "Handling special cases"
...
-    for project in "${PROW_TRUSTED_BUILD_CLUSTER_PROJECTS[@]}"; do
-        empower_ksa_to_svcacct \
-            "${project}.svc.id.goog[test-pods/k8s-infra-gcr-vuln-dashboard]" \
-            "${PROD_PROJECT}" \
-            $(svc_acct_email "${PROD_PROJECT}" "${VULN_DASHBOARD_SVCACCT}")
-    done
-
-    # Special case: grant the k8s-infra-gcr-vuln-dashboard account access to
-    # write to the prod vuln dashboard GCS bucket.
-    ensure_prod_gcs_bucket "${PROD_PROJECT}" "gs://${PROD_PROJECT}-vuln-dashboard" 2>&1 | indent
```
Comment (spiffxp) [inline infra/gcp/ensure-prod-storage.sh:392]: this has been manually removed already

File: infra/gcp/ensure-prod-storage.sh [hunk 2]
```diff
@@ -372,44 +372,6 @@ color 6 "Handling special cases"
             "${PROD_PROJECT}" \
             $(svc_acct_email "${PROD_PROJECT}" "${PROMOTER_VULN_SCANNING_SVCACCT}")
     done
-
-    color 6 "Ensuring prod promoter vuln-dashboard svcacct exists"
-    ensure_service_account \
-        "${PROD_PROJECT}" \
-        "${VULN_DASHBOARD_SVCACCT}" \
-        "k8s-infra container image vuln dashboard"
```
Comment (spiffxp) [inline infra/gcp/ensure-prod-storage.sh:380]: will remove this post-merge

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

File: infra/gcp/ensure-prod-storage.sh (hunk 1 above)
Comment (spiffxp) [reply infra/gcp/ensure-prod-storage.sh:392]: removed

File: infra/gcp/ensure-prod-storage.sh (hunk 2 above)
Comment (spiffxp) [reply infra/gcp/ensure-prod-storage.sh:380]: removed

**Comment by spiffxp** [comment]
Running ./ensure-prod-storage.sh

**Pull request merged**